ee/api/rbac/access_control.py

```diff
@@ -215,18 +215,28 @@ def _update_access_controls(self, request: Request, is_global=False):
 
         return Response(serializer.data, status=status.HTTP_200_OK)
 
-    @extend_schema(exclude=True)
-    @action(methods=["GET", "PUT"], detail=True)
-    def access_controls(self, request: Request, *args, **kwargs):
-        if request.method == "PUT":
-            return self._update_access_controls(request)
-
+    @extend_schema(
+        description="Returns access control settings for a specific resource, including user permissions, available access levels, and the default access level.",
+    )
+    @action(methods=["GET"], detail=True, required_scopes=["access_control:read"], url_path="access_controls/get")
```

Comment:
I don't like this, we should find another way without changing the paths. These endpoints are standardized and used heavily on the client side so this would break a bunch of this.